README.md

```diff
@@ -1,49 +1,48 @@
 # Welcome to Rails
 
-Rails is a web-application framework that includes everything needed to
-create database-backed web applications according to the
+Rails as a web-application framework includes everything needed to
```

Comment:
Opening with a long pronominal clause is harder to read. Current version looks better to me here.